.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
I think we should move from `master` to `main` in order to trigger this.

Reply:
I've requested to @dlorenc to rename the default branch to `main` :)

Reply:
ok branch renamed :) thanks Dan